recipes/wildmagic/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "WildMagic" %}
+{% set version = "5.14" %}
+{% set version_major = "5" %}
+{% set version_minor = "14" %}
+{% set sha256 = "951330bca2328d537dac2459bf154845295b8c4576a0adca5ffbf31b5a778d80" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}{{ version_major }}p{{ version_minor }}.zip
```

Comment:
FYI this is just the name the file is save as. It doesn't have to match where it was downloaded from.